packages/algob/src/internal/deployer.ts

```diff
@@ -603,19 +648,41 @@ export class DeployerDeployMode extends DeployerBasicMode implements Deployer {
 
   /**
    * This function will send Algos to ASC account in "Contract Mode"
-   * @param name     - ASC filename
+   * @param fileName     - ASC filename
    * @param flags    - Deployments flags (as per SPEC)
    * @param payFlags - as per SPEC
    * @param scTmplParams: Smart contract template parameters (used only when compiling PyTEAL to TEAL)
    */
   async fundLsig (
```

Comment:
same as above - maybe we should do a breaking change and use name everywhere. Let's chat on Discord. I will also ask the community.

Reply:
needs more discussion (breaking change)